src/actions/measurements.js

```diff
@@ -95,45 +95,38 @@ const getCollectionDisplayControls = (controls, collection) => {
   return newControls;
 };
 
-export const loadMeasurements = (json) => (dispatch, getState) => {
-  // TODO: Load controls from state to get potential url query parameters
-  const { tree, controls } = getState();
-  if (!tree.loaded) {
-    throw new Error("tree not loaded");
-  }
-
+export const parseMeasurementsJSON = (json) => {
   const collections = json["collections"];
   if (!collections || collections.length === 0) {
     throw new Error("Measurements JSON does not have collections");
   }
-
   collections.forEach((collection) => {
     /**
      * Keep backwards compatibility with single value threshold.
-     * Make sure thresholds are an array of values so that we don't have to
-     * check the data type in the D3 drawing process
-     * `collection.thresholds` takes precedence over the deprecated `collection.threshold`
-     */
+      * Make sure thresholds are an array of values so that we don't have to
+      * check the data type in the D3 drawing process
+      * `collection.thresholds` takes precedence over the deprecated `collection.threshold`
+      */
```

Comment:
Unexpected indent change here (and in following comments as well).

Reply:
Ah, thanks for catching this!